### ISSUE TITLE
Bad Performance on Office-Home, ImageCLEF-DA 

### ISSUE THREAD

**Issue by JingWang18** [kgl-prml/Contrastive-Adaptation-Network-for-Unsupervised-Domain-Adaptation#4]
Hi, 

I just tested your code on Office-Home and ImageCLEF-DA. The performance is not good. Any suggestions on how to solve this? Perhaps your proposed method is overfitting on VisDA and Office-31.

**Comment by luhaifeng19947** [comment]
> Hi,
> 
> I just tested your code on Office-Home and ImageCLEF-DA. The performance is not good. Any suggestions on how to solve this? Perhaps your proposed method is overfitting on VisDA and Office-31.

**Comment by luhaifeng19947** [comment]
@JingWang18 Yes, I also found it overfitting. The main reason is that Office-31  is not splitted into test set and validate set. CAN can do very well on  validate set, but show bad performance on test set. I found dropout can alleviate the problem.

**Comment by GlassyWing** [comment]
@JingWang18 Could you share the test results of these two data sets? Thank you
